keras_retinanet/bin/train.py

```diff
@@ -451,8 +458,21 @@ def main(args=None):
         args = sys.argv[1:]
     args = parse_args(args)
 
+    img_path = list(csv.reader(open(args.annotations), delimiter=','))[0][0]
+    img_path = os.path.join(os.path.dirname(args.annotations), img_path)
+    if rasterio.open(img_path).count == 3 and args.channels == 4:
+        raise ValueError('Images have only 3 channels!')
+
+    if args.stat is None and args.channels == 4:
```

Comment:
```suggestion
    if args.stat is None and args.channels != 3:
```
It would be better if we'll add possibility to work with additional bands.